internal/logger/logger.go

```diff
@@ -0,0 +1,30 @@
+package logger
+
+import (
+	"log/slog"
+	"os"
+	"strings"
+)
+
+func AppLogger(levelStr string) *slog.Logger {
+	var level slog.Level
+	switch strings.ToLower(levelStr) {
+	case "debug":
+		level = slog.LevelDebug
+	case "info":
+		level = slog.LevelInfo
+	case "warn":
+		level = slog.LevelWarn
+	case "error":
+		level = slog.LevelError
+	default:
+		level = slog.LevelInfo
+	}
+
+	handler := slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{
+		Level: level,
+	})
+
+	logger := slog.New(handler)
+	return logger
+}
```

Comment:
Be nice to validate the given log level (current implementation has this): [see what we do in `generate-policy-bot-config` for that](https://github.com/grafana/generate-policy-bot-config/blob/0a7b68dcbcdcbc441ea72ad961daa1fd38299fba/cmd/generate-policy-bot-config/main.go#L89-L100).

Also (optional): see our `setupLogger` in the same file. There are some handlers we can use to make the output a bit nicer than the `TextHandler`.